pipeline/test/services/service-cluster/testOpensearch.sh

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+# (return 0 2>/dev/null) && sourced=1 || sourced=0
+
+INNER_SCRIPTS_PATH="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+source "${INNER_SCRIPTS_PATH}/../funcs.sh"
```

Comment:
In that file change the check on `$PIPELINE` variable to reference it with `${PIPELINE-}` as it not set when run via this script.